scripts/run.sh

```
@@ -0,0 +1,31 @@
+# start ros bridge
+echo "[INFO] Starting ros bridge..."
+./build/Release/vola_ros_bridge &
+CTRL_PID=$!
+
+# Start controller node
+echo "[INFO] Launching Lee controller..."
+./build/Release/lee_control_node &
+CTRL_PID=$!
+
+# === CLEANUP HANDLER ===
+trap "echo; echo '[INFO] Shutting down...'; kill $VOLASIM_PID $CTRL_PID $RVIZ_PID; wait" SIGINT SIGTERM
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)